tools/serverpod_cli/lib/src/util/string_manipulation.dart

```diff
@@ -20,6 +27,26 @@ List<String> splitIgnoringBrackets(
         depth++;
       } else if (char == '>' || char == ')') {
         depth--;
+      } else if (char == "'" && !insideDoubleQuotes) {
+        if (insideSingleQuotes) {
+          // Check if it's an escaped single quote
+          if (i > 0 && input[i - 1] == '\\') {
```

Comment:
Question: Won't `input[i - 1]` only retrive a single character? How can that match the two `\\` characters? Do we have tests for this?

Reply:
The first \ escapes the second \